packages/roosterjs-editor-types/lib/enum/BulletListType.ts

```diff
@@ -32,4 +37,9 @@ export const enum BulletListType {
      * Bullet triggered by =>
      */
     UnfilledArrow,
+
+    /**
+     * Maximum value of the enum
+     */
+    Max,
```

Comment:
This will cause Max has a different number value than UnfilledArrow, but it is still a valid value. What if people call an API and pass in Max or Min? There will not be build time error but runtime will show incorrect result.

Reply:
That's true, I fixed. I used max -1 to validate the number. Now I'm using the last max equal to the last number value of the enum